packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/Comments/Dialog/index.tsx

```diff
@@ -135,12 +135,11 @@ export const Dialog: React.FC = () => {
                 setEditing={setEditing}
                 hasReplies={replies.length}
               />
-              {replies.length ? (
```

Comment:
I set this because of `[null]` that the replies would break when you deleted the last one

Reply:
no worries, this is now handled inside the component (so it doesn't break): https://github.com/codesandbox/codesandbox-client/pull/3813/files#diff-6b9465e97e2db14cb6ef5c5076e3b42bR419

The reason for moving it in is to handle the initial state of "no replies" inside the component as the first step to the subsequent states of loading replies / transitions

Reply:
Goood